### PR TITLE
Bump RS

### PR DESCRIPTION
**Pull request by David-Henner**
## What's new in this PR?

Bumped Request Strategy and MockTransport

### CODE REVIEW

**Comment by zenkins** [comment]
Can one of the admins verify this patch?

File: Cartfile.resolved [hunk 1]
```diff
@@ -1,19 +1,19 @@
 github "wireapp/HTMLString" "4.1.0-beta.1-xcode_10_2"
 github "wireapp/PINCache" "2.3-swift3.1"
 github "wireapp/ZipArchive" "v2.1.3"
-github "wireapp/avs-ios-binaries" "5.3.194"
+github "wireapp/avs-ios-binaries" "5.3.195"
```
Comment (David-Henner) [inline Cartfile.resolved]: unsure if this should be included

Comment (billypchan) [reply Cartfile.resolved]: yes, but did 5.4 released?

Comment (David-Henner) [reply Cartfile.resolved]: no, `5.3.195` is the latest

**Comment by billypchan** [comment]
@zenkins release major

**Pull request merged**